assets/scss/_base.scss

```
@@ -65,8 +65,8 @@ footer {
   .button {
     display: inline-block;
     border-radius: 6px;
-    padding: 0 20px;
-    line-height: 40px;
+    padding: 8px 20px;
+    line-height: 1.5rem;
```

You could go even narrower:

```suggestion
    line-height: 1.3rem;
```

---

Take care when setting `line-height` separately from `font-size` or using different units.
(I realize this is a drawback in _master_ as well).

---

That's right. I'd like to keep this unit now but it would be nice to replace line-height in this file should be replaced relative ones in the future.